### PR TITLE
Feature/gwdc 74 different warehouses documentation

### PR DESCRIPTION
**Pull request by Martin-Welte**
# TL;DR

- adds documentation for GBW shipping method
- adds documentation for Plugin settings Shipping Details
- enables search within docsify

### CODE REVIEW

File: docs/gbw-shipping-method.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# GBW Shipping Method
+
+## Setup
+
+---
+> The `GBW Shipping Method` will only be used if enabled in the [Plugin settings](./setup#settings-tab-shipping-details), otherwise all orders will be sent to Gebrüder Weiss.
+---
+
+The Plugin offers a custom shipping method that can be used for defining custom shipping zones and custom shipping costs. 
+How you can generally define shipping zone you can read here: [Setting up Shipping Zones](https://woocommerce.com/document/setting-up-shipping-zones/)
```
Comment (philipp-hemmer) [inline docs/gbw-shipping-method.md]: i'd write `How shipping zones are defined is documented here`

File: docs/gbw-shipping-method.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Once you have added the shipping Method, you can configure this method by clicking `edit` on the shipping method.
+
+![woocommerce-shipping-edit-gbw-shipping.png](./assets/images/woocommerce-shipping-edit-gbw-shipping.png ':size=900')
+
+![woocommerce-shipping-add-warhouse-id](./assets/images/woocommerce-shipping-add-warhouse-id.png ':size=900')
+
+*Figure 2 & 3: Edit GBW Shipping Method and add Gebrüder Weiss Warehouse ID*
+
+In the overlay you can add a specific Gebrüder Weiss Warehouse ID. This is only relevant if you are a customer who has stock in different warehouses from Gebrüder Weiss. You can configure your shipping zones to use the nearest Warehouse by suppling the nearest warehouse id within the shipping zone.
```
Comment (philipp-hemmer) [inline docs/gbw-shipping-method.md]: supplying

File: docs/gbw-shipping-method.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+![woocommerce-shipping-add-warhouse-id](./assets/images/woocommerce-shipping-add-warhouse-id.png ':size=900')
+
+*Figure 2 & 3: Edit GBW Shipping Method and add Gebrüder Weiss Warehouse ID*
+
+In the overlay you can add a specific Gebrüder Weiss Warehouse ID. This is only relevant if you are a customer who has stock in different warehouses from Gebrüder Weiss. You can configure your shipping zones to use the nearest Warehouse by suppling the nearest warehouse id within the shipping zone. 
+If no Warehouse ID is supplied, the default of your account will be used. Please Contact Gebrüder Weiss Support if you have further Questions about this. 
+
+## Shipping Rate
+
+Per default the `GBW Shipping Method` does not have additional price defined. If you want to override the default shipping rate calculation from the Plugin you can use the filter `gbw_shipping_rate` as follows:
```
Comment (philipp-hemmer) [inline docs/gbw-shipping-method.md]: has no additional price defined

**Review by philipp-hemmer — CHANGES_REQUESTED** [review]

File: docs/gbw-shipping-method.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+Once you have added the shipping Method, you can configure this method by clicking `edit` on the shipping method.
+
+![woocommerce-shipping-edit-gbw-shipping.png](./assets/images/woocommerce-shipping-edit-gbw-shipping.png ':size=900')
+
+![woocommerce-shipping-add-warhouse-id](./assets/images/woocommerce-shipping-add-warhouse-id.png ':size=900')
+
+*Figure 2 & 3: Edit GBW Shipping Method and add Gebrüder Weiss Warehouse ID*
+
+In the overlay you can add a specific Gebrüder Weiss Warehouse ID. This is only relevant if you are a customer who has stock in different warehouses from Gebrüder Weiss. You can configure your shipping zones to use the nearest Warehouse by supplying the nearest warehouse id within the shipping zone. 
+If no Warehouse ID is supplied, the default of your account will be used. Please Contact Gebrüder Weiss Support if you have further Questions about this.
```
Comment (mattcrn) [inline docs/gbw-shipping-method.md]: contact lowercase

**Review by philipp-hemmer — APPROVED** [review]

**Review by mattcrn — APPROVED** [review]
1 little typo but approved

**Pull request merged**